src/evaluate.cpp

```diff
@@ -67,84 +75,92 @@ std::string currentEvalFileName = "None";
 // variable to have the engine search in a special directory in their distro.
 void NNUE::init() {
 
-    std::string eval_file = std::string(Options["EvalFile"]);
-    if (eval_file.empty())
-        eval_file = EvalFileDefaultName;
+    for (NetSize netSize : {Big, Small})
+    {
+        std::string eval_file = std::string(Options[EvFiles[netSize]]);
```

Comment:
btw this is the reason right now why the CI files, `Options[]` default constructs another entry to the uci options.